src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -20,7 +21,7 @@
       <v-flex class="info-text" xs7 v-else>
         No name translations
       </v-flex>
-      <v-flex xs2 class="align-right" v-if="!hasNameTranslationChange">
+      <v-flex xs2 class="align-right" v-if="!hasNameTranslationChange && !isSummaryMode">
```

Comment:
While you're in here, could you please replace the deprecated v-layout and v-flex elements with v-rows and v-cols?

Reply:
Done